src/main/java/io/vertx/json/schema/Draft.java

```diff
@@ -25,30 +25,43 @@ public enum Draft {
   /**
    * Draft 4 - <a href="http://json-schema.org/draft-04/schema#">http://json-schema.org/draft-04/schema#</a>
    *
-   * Usually used by Swagger 2.0
+   * Usually used by OpenAPI 3.0
```

Comment:
Not sure about this, I am under the impression that openapi 3.0.x requires draft-7

Reply:
See https://github.com/OAI/OpenAPI-Specification/blob/main/schemas/v3.0/schema.json#L3
This is why I thought it requires draft-4